cmake/Modules/FindeSpeak.cmake

```diff
@@ -3,13 +3,12 @@ include(ToolchainLibraryFinder)
 # Find ALSA for eSpeak
 find_package(ALSA REQUIRED)
 
-# Find PortAudio for eSpeak
-find_package(PortAudio REQUIRED)
+find_package(pcaudio REQUIRED)
```

Comment:
How is this used?

Reply:
Well spotted. Will make a PR to remove that.

Reply:
#720

Reply:
But you added `pcaudio` to replace `PortAudio` in this PR? Why make another PR that removes `PortAudio` and doesn't address the question of how `pcaudio` is used?

Reply:
it would also appear that pcaudio is a dependency of espeawould also seem that pcaudio is a dependency of espeak, so removing it seems like a bad idea

Reply:
`pcaudio` which replaces `PortAudio` isn't actually used. They refer to the same functionality/are meant to do the same thing, which we apparently don't need

Reply:
we use it transitively via espeak, so I think we should still be building it we just dont need to `find_package` it (unless we want the sanity check that it is part of our toolchain)

Reply:
Trueeeee. [looks like you should have it hahaha](https://github.com/NUbots/NUbots/pull/691#discussion_r713786179). Will close that PR. 


On a related note, should we be linking `pcaudio` explicitly with `target_link_libraries`?

Reply:
> On a related note, should we be linking pcaudio explicitly with target_link_libraries?

It shouldn't be necessary. Linking against `espeak-ng` should cause any necessary symbols to be loaded from `pcaudio` (assuming there are any at link time). The more likely behaviour will be the loading of `pcaudio` at runtime when we load `espeak-ng`. The only reason we would want to link against `pcaudio` explicitly is as a "sanity check" on its existence (which is the only role that `find_package` is playing in this instance). I think we should just remove the `find_package` line and the find module, but keep it in the `Dockerfile`